### PR TITLE
Cache seeder telemetry

### PR DESCRIPTION
**Pull request by antrix1989**

### CODE REVIEW

File: IdentityCore/src/MSIDExternalAADCacheSeeder.m [hunk 1]
```diff
@@ -69,9 +73,17 @@ - (void)seedTokenResponse:(MSIDTokenResponse *)originalTokenResponse
     
     MSID_LOG_WITH_CTX(MSIDLogLevelInfo, requestParameters, @"Beginning external cache seeding.");
     
-    void (^completionBlockWrapper)(void) = ^
+    [[MSIDTelemetry sharedInstance] startEvent:requestParameters.telemetryRequestId eventName:MSID_TELEMETRY_EVENT_EXTERNAL_CACHE_SEEDING];
```
Comment (oldalton) [inline IdentityCore/src/MSIDExternalAADCacheSeeder.m]: does it make it into the aggregated blob?

File: IdentityCore/src/MSIDExternalAADCacheSeeder.m [hunk 2]
```diff
@@ -69,9 +73,17 @@ - (void)seedTokenResponse:(MSIDTokenResponse *)originalTokenResponse
...
+    
+    void (^completionBlockWrapper)(BOOL success) = ^(BOOL success)
     {
         MSID_LOG_WITH_CTX(MSIDLogLevelInfo, requestParameters, @"External cache seeding finished.");
+
```
Comment (oldalton) [inline IdentityCore/src/MSIDExternalAADCacheSeeder.m]: I'd also like to track if we made network request or got id token from cache to catch regressions where we always make 2 network requests.

File: IdentityCore/src/MSIDExternalAADCacheSeeder.m (hunk 1 above)
Comment (antrix1989) [reply IdentityCore/src/MSIDExternalAADCacheSeeder.m]: Updated

File: IdentityCore/src/MSIDExternalAADCacheSeeder.m (hunk 2 above)
Comment (antrix1989) [reply IdentityCore/src/MSIDExternalAADCacheSeeder.m]: done

**Pull request merged**